docs/ref/urlresolvers.txt

```diff
@@ -215,7 +215,7 @@ view would raise a ``Http404`` error before redirecting to it::
         # modify the request and response as required, e.g. change locale
         # and set corresponding locale cookie
 
-        view, args, kwargs = resolve(urlparse(next)[2])
+        view, args, kwargs = resolve(urlsplit(next)[2])
```

Comment:
Since we are changing this bit, could we also use `.path`  for clarity?
```suggestion
        view, args, kwargs = resolve(urlsplit(next).path)
```